src/containers/MyNdla/ArenaCards/ArenaCard.tsx

```diff
@@ -0,0 +1,205 @@
+    svg:nth-child(2) {
+      display: block;
+    }
+  }
+`;
+
+const StyledTopicCard = css`
+  background-color: ${colors.background.lightBlue};
+  &:hover,
+  &:focus-visible {
+    background-color: ${colors.brand.lighter};
+  }
+`;
+
+const StyledCardContainer = styled(SafeLink)`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: ${spacing.normal};
+  padding-right: ${spacing.medium};
+  border: 1px solid ${colors.brand.light};
+  border-radius: ${misc.borderRadius};
+  box-shadow: none;
+  > * > span {
+    text-decoration: underline;
+  }
+  &:hover,
+  &:focus-visible {
+    background-color: ${colors.brand.lighter};
+    > * > span {
+      text-decoration: none;
+    }
+  }
+`;
+
+const StyledTextContainer = styled.div`
+  margin-right: auto;
+  display: flex;
+  flex-direction: column;
+`;
+
+const StyledHeader = styled.span`
+  ${fonts.sizes('18px', '24px')};
+  font-weight: ${fonts.weight.semibold};
+  color: ${colors.brand.primary};
+  margin: 0;
+`;
+
+const StyledDescriptionText = styled.div`
+  padding-top: ${spacing.xsmall};
+  ${fonts.sizes('16px', '26px')};
+  ${mq.range({ until: breakpoints.mobileWide })} {
+    display: none;
+  }
+`;
+
+const StyledText = styled.div`
+  margin: 0;
+  color: ${colors.text.primary};
+  padding-top: ${spacing.xsmall};
+  ${fonts.sizes('16px', '26px')};
+  font-weight: ${fonts.weight.normal};
+`;
+
+const StyledCountContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: ${colors.text.primary};
+  ${mq.range({ until: breakpoints.tabletWide })} {
+    display: none;
+  }
+`;
+
+const StyledCountDiv = styled.div`
+  ${fonts.sizes('18px', '24px')};
+  font-weight: ${fonts.weight.semibold};
+`;
+
+const StyledLeftIcon = styled(Icon)`
+  margin-right: ${spacing.normal};
+  min-width: 40px;
+  height: 40px;
+  color: ${colors.brand.primary};
+  ${mq.range({ until: breakpoints.mobileWide })} {
+    display: none;
+  }
+`;
+
+const StyledLockedIcon = styled(Icon)`
+  width: 24px;
+  height: 24px;
+  color: ${colors.brand.primary};
+`;
+
+const FolderFilledIcon = StyledLeftIcon.withComponent(Forum);
+const FolderOutlinedIcon = StyledLeftIcon.withComponent(ForumOutlined);
+const LockedIcon = StyledLockedIcon.withComponent(Locked);
+
+const StyledAvatarContainer = styled.div`
+  //Placeholder until Avatar
+  margin-right: ${spacing.normal};
+  width: 48px;
+  height: 48px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: ${colors.white};
+  border: solid 1px ${colors.brand.tertiary};
+  border-radius: 50px;
+  ${mq.range({ until: breakpoints.mobileWide })} {
+    display: none;
+  }
+`;
+
+const ArenaCard = ({
+  id,
+  cardType,
+  title,
+  subText,
+  timestamp,
+  count,
+  locked,
+}: Props) => {
+  const { t } = useTranslation();
+  return (
+    <StyledCardContainer
+      id={id}
+      css={cardType === 'ArenaCategory' ? StyledCategoryCard : StyledTopicCard}
+      to="topic"
```

Comment:
Om du linker til kategorien du får fra backend så kan du plukke opp den i TopicPage
```suggestion
      to={`/myndla/arena/${id}`}
```